mint-auditor/src/bin/main.rs

```diff
@@ -342,6 +344,31 @@ fn audit_block_data(
         };
     }
 
+    // Audit burns.
+    for burn_tx_out in &sync_block_data.burn_tx_outs {
+        match AuditedBurn::attempt_match_burn_with_withdrawal(burn_tx_out, config, conn) {
```

Comment:
nit: `attempt` -> `try`, matching style on like 325 (unless there is a high-level difference between them that I'm missing)